truckersmp_cli/main.py

```diff
@@ -138,11 +149,23 @@ def main():
 
     # start truckersmp with proton or wine
     if Args.start:
-        # check for Proton availability when starting with Proton
-        if (Args.proton
-                and not os.access(os.path.join(Args.protondir, "proton"), os.R_OK)):
-            sys.exit("""Proton is not found in {}
+        if Args.proton:
+            # check for Proton availability when starting with Proton
+            if not os.access(os.path.join(Args.protondir, "proton"), os.R_OK):
+                sys.exit("""Proton is not found in {}
 Run with '--update' option to install Proton""".format(Args.protondir))
+            # check for Steam Runtime availability and the permission of "var"
+            # when starting with Proton + Steam Runtime
+            run = os.path.join(Args.steamruntimedir, "run")
+            var = os.path.join(Args.steamruntimedir, "var")
+            if not Args.without_steam_runtime:
+                if not os.access(run, os.R_OK | os.X_OK):
+                    sys.exit("""Steam Runtime is not found in {}
+Update game with '--use-steam-runtime' option to install Steam Runtime""".format(
+                             Args.steamruntimedir))
+                if not os.access(var, os.R_OK | os.W_OK | os.X_OK):
+                    sys.exit("""The "var" directory in Steam Runtime
+({}) must be writable""".format(var))
```

Comment:
I started over fresh without the runtime and after downloading and trying to start this kicks in. The folder isn't present directly after creating so we may want to create it.

I did a 'mkdir' for that folder and now I get:
~~~
[…]
** INFO **  Using Steam Runtime container
** DEBUG **  Shared paths: ['/mnt/gamehub/native2proton/Euro Truck Simulator 2/', '/home/lucki/.local/share/truckersmp-cli/Proton', '/home/lucki/.local/share/truckersmp-cli/Euro Truck Simulator 2/prefix']
Failed to run wineboot: [Errno 2] Datei oder Verzeichnis nicht gefunden: '--'
~~~